python/variable.cpp

```diff
@@ -211,7 +197,6 @@ of variances.)");
         return size_of(self, SizeofTag::Underlying);
       });
 
-  bind_init_list(variable);
   // Order matters for pybind11's overload resolution. Do not change.
   bind_init_0D_numpy_types(variable);
   bind_init_0D_native_python_types<bool>(variable);
```

Comment:
Are these still reachable now that the constructor above accepts any `py::object`?

Reply:
Yes. These are for 0d and have a different signature (no `dim` and `value` instead of `values`).